addons/calendar/static/src/views/calendar_form/calendar_quick_create.js

```diff
@@ -45,6 +45,15 @@ export class CalendarQuickCreateFormController extends CalendarFormController {
         const context = getDefaultValuesFromRecord(this.model.root.data)
         this.props.goToFullEvent(context);
     }
+
+    async onRecordSaved() {
```

Comment:
Seems quite hackish and flow-specific, don't we have better way of managing that ? Do we need it anyway ? Any flow-specific code makes activity management harder I think.

Reply:
Feels like it is fragile. Indeed it completely depends on the breadcrumb, which is not something you can really count on. If we go that way, we should be sure it exists and leads to the right record.

Moreover it depends on context, which means it is already not reachable unless the action prepares it. So I suppose we can try a more classic solution, with a specific view for that specific flow, instead of a JS hack, yes. Maybe check both and compare the diff :) .